### ISSUE TITLE
Deploy disaster response web app 

### ISSUE THREAD

**Issue by chris-hoffmann** [chris-hoffmann/disaster_response#1]
__Option 1__: use a logistic regression classifier 
This yields to smaller more memory efficient model since we don't need to save parameters for 200 decision trees. 

~~__Option 2__:  keep Random Forest classifier and deploy via Google App Engine~~

__DoD__
- [ ] deploy as Heroku app
- [ ] add command line parsing to training script

__References__
- https://fallfordata.com/multi-label-classification-of-texts-with-nltk/
- https://www.kaggle.com/roccoli/multi-label-classification-with-sklearn
